demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -292,6 +292,82 @@ def _collect_playbooks_dependencies(pack_playbooks, id_set):
 
         return dependencies_packs
 
+    @staticmethod
+    def _collect_layouts_dependencies(pack_layouts, id_set):
+        """
+        Collects layouts pack dependencies.
+
+        Args:
+            pack_layouts (list): collection of pack playbooks data.
+            id_set (dict): id set json.
+
+        Returns:
+            set: dependencies data that includes pack id and whether is mandatory or not.
+
+        """
+        dependencies_packs = set()
+
+        for layout in pack_layouts:
+            layout_data = next(iter(layout.values()))
+
+            related_incident_and_indicator_types = layout_data.get('incident_and_indicator_types', [])
+            packs_found_from_incident_indicator_types = PackDependencies._search_packs_by_items_names(
+                related_incident_and_indicator_types, id_set['IncidentTypes'] + id_set['IndicatorTypes'])
+
+            if packs_found_from_incident_indicator_types:
+                pack_dependencies_data = PackDependencies. \
+                    _label_as_mandatory(packs_found_from_incident_indicator_types)
```

Comment:
Do we want to label all found results as mandatory dependency or there can be cases where incident type/indicator type are generic? Similar to reputations commands in integrations